docs/sdks/uid2-sdk-ref-java.md

```diff
@@ -63,56 +58,165 @@ The binary is published on the Maven repository:
+| `SUCCESS`                       | The raw UID2 was successfully encrypted and a UID2 token was returned. |
+| `NOT_AUTHORIZED_FOR_KEY`        | The requester does not have authorization to use the encryption key.   |
+| `NOT_AUTHORIZED_FOR_MASTER_KEY` | The requester does not have authorization to use the master key.       |
+| `NOT_INITIALIZED`               | The client library is waiting to be initialized.                       |
+| `KEYS_NOT_SYNCED`               | The client has failed to synchronize keys from the UID2 service.       |
+| `ENCRYPTION_FAILURE`            | A generic encryption failure occurred.                                 |
+
+### Decryption Response Content
+
+Whether decrypting with the `BidstreamClient` or the `SharingClient`, the SDK returns the information shown in the following table.
+
+| Property      | Description                                                                                                                                     |
+|:--------------|:------------------------------------------------------------------------------------------------------------------------------------------------|
+| `status`      | The decryption result status. For a list of possible values and definitions, see [Decryption Response Statuses](#decryption-response-statuses). |
+| `uid`         | The raw UID2 for the corresponding UID2 token.                                                                                                  |
+| `established` | The timestamp indicating when a user first established the UID2 with the publisher.                                                             |
+
+### Decryption Response Statuses
+
+Decryption response codes, and their meanings, are shown in the following table.
+
+| Value                      | Description                                                             |
+|:---------------------------|:------------------------------------------------------------------------|
+| `SUCCESS`                  | The UID2 token was decrypted successfully and a raw UID2 was returned.  |
+| `NOT_AUTHORIZED_FOR_KEY`   | The requester does not have authorization to decrypt this UID2 token.   |
+| `NOT_INITIALIZED`          | The client library is waiting to be initialized.                        |
+| `INVALID_PAYLOAD`          | The incoming UID2 token is not a valid payload.                         |
+| `EXPIRED_TOKEN`            | The incoming UID2 token has expired.                                    |
+| `KEYS_NOT_SYNCED`          | The client has failed to synchronize keys from the UID2 service.        |
+| `VERSION_NOT_SUPPORTED`    | The client library does not support the version of the encrypted token. |
+| `INVALID_TOKEN_LIFETIME`   | The token has invalid timestamp.        |
+
+## Usage for DSPs
+
+
+The following instructions provide an example of how you can decode bid stream tokens using the UID2 SDK for Java as a DSP.
```

Comment:
Suggest turn it around:

The following instructions provide an example of how a DSP can decode bid stream tokens using the UID2 SDK for Java.
or
The following instructions provide an example of how you, as a DSP, can decode bid stream tokens using the UID2 SDK for Java.